domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -142,6 +143,8 @@ public class PostgresClient {
   /** map (Vertx, String tenantId) to PostgresClient */
   private static final MultiKeyMap<Object, PostgresClient> CONNECTION_POOL =
       MultiKeyMap.multiKeyMap(new HashedMap<>());
+  private static final MultiKeyMap<Object, PostgresClient> CONNECTION_POOL_READER =
+      MultiKeyMap.multiKeyMap(new HashedMap<>());
```

Comment:
By default CONNECTION_POOL_READER should equal CONNECTION_POOL so that we have a single pool as before. Only when there's reader settings in the configuration the second pool should be created.

Reply:
Or by default CONNECTION_POOL_READER should be null, and if null clientReader is set to client.